api/graphql/index.js

```diff
@@ -257,6 +261,8 @@ export function makeMutations (sessionId, userId, isAdmin) {
 
     deleteGroupTopic: (root, { id }) => deleteGroupTopic(userId, id),
 
+    deleteMe: (root, {id}) => deleteUser({ sessionId, userId: id }),
```

Comment:
i just noticed that you are passing in id here, shouldnt you just always use userId? we dont want to let people delete other users, at least through this deleteMe call

Reply:
Ahh right. I noticed that userId was the account I was logged into, and I didn't want to delete that account, so changed this. Will change it back